src/late_multicellular_stage/MulticellularHUD.tscn

```diff
@@ -367,7 +395,6 @@ grow_horizontal = 1
 grow_vertical = 1
 
 [node name="ProcessPanel" parent="." instance=ExtResource("13")]
-anchors_preset = 0
```

Comment:
Anchor changes also seem suspicious as these popups are pretty picky now in what their anchors are...

Reply:
Is it really necessary for `anchors_preset = 0` to be here? Appearently Godot really wants it to be gone as every time I save some change it is gone. I can revert it with notepad (IDK what preset it actually is) but it would be tedious to do when saving each time. It might be related that I have Godot opened with Compatibility mode because I need to use OpenGL otherwise I'd need to have quite many attempts at opening Godot only